datadog/resource_datadog_dashboard_test.go

```diff
@@ -452,6 +452,56 @@ resource "datadog_dashboard" "ordered_dashboard" {
 }`, uniq)
 }
 
+func datadogSimpleOrderedDashboardConfig(uniq string) string {
+	return fmt.Sprintf(`
+resource "datadog_dashboard" "simple_ordered_dashboard" {
+	title         = "%s"
+	description   = "Created using the Datadog provider in Terraform"
+	layout_type   = "ordered"
+	is_read_only  = true
+	widget {
+		alert_graph_definition {
+			alert_id = "895605"
```

Comment:
Should this be a random value like `123456` ?

Reply:
isn't 895605 random ?

Reply:
Sorry, I meant fake. Wasn't sure if this was a real id or not 🙂

Reply:
well, it's not exactly sensitive anyway. Plus I just c/p it from the other dashboard defs

Reply:
pretty sure 123456 is also a real id ;)

Reply:
hahaha, fair enough :)